### ISSUE TITLE
FullDebug - Gcc Pipeline 

### ISSUE THREAD

**Issue by roigcarlo** [KratosMultiphysics/Kratos#11564]
Dear @KratosMultiphysics/all You may have noticed that the `Ubuntu - FullDebug/Gcc` CI is giving problems for a couple of days.

After some investigation we have finally find the cause ( insufficient disk space ) and we are working to solve it. Exceptionally we have set that CI as not required for merge, so if its the only one failing in your PR you should be able to merge it.

Once this is resolved we will re-enable de requirement for that workflow as normal.

**Comment by jcotela** [comment]
It looks like there may be a similar problem in FullDebug/clang too, I am getting this error
```
 CMake Error at applications/IgaApplication/cmake_install.cmake:256 (file):
  file INSTALL cannot copy file
  "/__w/Kratos/Kratos/build/FullDebug/applications/IgaApplication/libKratosIgaCore.so"
  to "/__w/Kratos/Kratos/bin/FullDebug/libs/libKratosIgaCore.so": No space
  left on device.
```
For this build job 
https://github.com/KratosMultiphysics/Kratos/actions/runs/6535183228/job/17809984790?pr=11059

I tried rerunning the job and it keeps failing on the build step, so it seems a persistent issue.

**Comment by roigcarlo** [comment]
Yep, Both FullDebug pipelines are broken because we exceed the disk quota. 

Since there is nothing left to clean i am preparing a new compilation script that incrementally releases compilation artifacts, but its not ready yet. Until its done it will keep failing. 

Details on #11566
